MAESTRO/Snakemake/scATAC/Snakefile

```diff
@@ -200,10 +154,21 @@ if config["platform"] == "microfluidic":
             "Result/Benchmark/%s_QCMerge.benchmark" %(config["outprefix"])
         shell:
             "python " + SCRIPT_PATH + "/scATAC_microfluidic_QC.py --log-dir {params.log} --directory {params.outdir};"
+    
+    rule scatac_celldedup:
```

Comment:
we need to make it as an option at config.yaml
see how I did it https://github.com/crazyhottommy/pyflow-single-cell/blob/feat/dedup/scATAC/rules/sc_atac_dedup.smk